xbmc/guilib/TextureGL.h

```diff
@@ -40,6 +40,6 @@ class CGLTexture : public CBaseTexture
 
 protected:
   GLuint m_texture = 0;
-  bool m_profile32 = false;
+  bool m_profile30 = false;
```

Comment:
@fritsch btw, a profile 3.0 does not exist. profiles were introduced with 3.2

Reply:
Thx should I change it to m_oglversion?